README_pypi.md

```diff
@@ -0,0 +1,227 @@
+
+Check out the paper at [SMARTS: Scalable Multi-Agent Reinforcement Learning Training School for Autonomous Driving](https://arxiv.org/abs/2010.09776) for background on some of the project goals.
+
+![](docs/_static/smarts_envision.gif)
+
+## Installation of SMARTS package
+You can install SMARTS package from [PyPI](https://pypi.org/project/smarts/).
+
+Package installation requires Python >= 3.7. 
+```bash
+# If you dont have python 3.7 or higher, make sure to install or update python first
+
+# For windows user 
+py -m pip install smarts
+ 
+# For Unix/MACOSX user
+python3 -m pip install smarts
+```
+## Installation of SUMO
+[SUMO or "Simulation of Urban Mobility"](https://sumo.dlr.de/docs/index.html) is an open source highly portable, microscopic and continuous traffic simulation package which SMARTS currently uses as a tool to create scenarios. You need to install SUMO to use the different modules and packages installed through the command above.
+You can find a general location for sources and binaries for all platforms here: 
+- https://sumo.dlr.de/docs/Downloads.php
+
+If you wish to compile SUMO yourself, the repository is located here: 
+ - https://github.com/eclipse/sumo.
+ - If you do so make sure to check out the [most current version of 1.7](https://github.com/eclipse/sumo/commits/v1_7_0) or higher.
+
+and the build instructions:  
+ - https://sumo.dlr.de/docs/Developer/Main.html#build_instructions
+
+### Linux
+
+SUMO primarily targets Ubuntu versions >= 16.04. So you may not be able to download pre-built binaries for SUMO 1.7 from a package manager if you're running another OS.
+
+If you try through a package manager make sure to command-line call `sumo` to make sure that you have the right version of SUMO.
+
+We would recommend using the prebuilt binaries but if you are using Ubuntu 16 (Xenial),there is a bash script in `extras/sumo/ubuntu_build' that you can use to automate the compilation of SUMO version 1.5.0.
+
+### macOS
+
+macOS installation of SUMO is straight-forward. See https://sumo.dlr.de/docs/Downloads.php#macos_binaries for details.
+
+* If you wish to find binaries: **[SUMO Download Page](https://sumo.dlr.de/docs/Downloads.php)**
+    * **Please note that building SUMO may not install other vital dependencies that SUMO requires to run.**
+    * If you build from the git repository we recommend you use: **[SUMO version 1.7.0](https://github.com/eclipse/sumo/commits/v1_7_0)** or higher
+  
+## Modules installed
+Through smarts package, you have access to ['core'](https://github.com/huawei-noah/SMARTS/tree/master/smarts/core), ['env'](https://github.com/huawei-noah/SMARTS/tree/master/smarts/env), ['sstudio'](https://github.com/huawei-noah/SMARTS/tree/master/smarts/sstudio) and ['zoo'](https://github.com/huawei-noah/SMARTS/tree/master/smarts/zoo) as sub-modules under 'smarts' module and 
```

Comment:
SMARTS should consistently be in all caps throughout this page (including above), _except_ for when you're referring to the 'smarts' module and/or folder, but that should be in back ticks, like `smarts` instead.
Also, we should use "the SMARTS package" in the above like "Through the SMARTS package".